src/notify/app/validators/request_validator.py

```diff
@@ -25,6 +25,24 @@ def verify_headers(headers: dict, api_key: str) -> tuple[bool, str]:
 
     return True, ""
 
+
+def verify_get_statuses_headers(headers: dict, api_key: str) -> tuple[bool, str]:
+    lc_headers = header_keys_to_lower(headers)
+    if lc_headers.get(API_KEY_HEADER_NAME) is None:
```

Comment:
are all these headers still required just to GET statuses?

Reply:
We don't currently do HMAC verification for GET statuses but we do for POST to create statuses.
I think we may end up with another story which refactors the checks so that the callback requests flow through a different set of checks from the Consumer request validations.